games/j.yaml

```
@@ -58,6 +58,24 @@
   url: https://bytonic.de/html/jake2.html
   info: Java port of the Quake2 game engine
 
+- name: Javascript BoulderDash
+  originals:
+  - Boulder Dash
+  type: clone
+  repo: 'https://github.com/jakesgordon/javascript-boulderdash/'
+  url: 'https://jakesgordon.com/writing/javascript-boulderdash/'
+  development: sporadic
```

```suggestion
  development: halted
```
the latest commit is just updating some links in the readme